pkg/policy/policy_file.go

```diff
@@ -264,6 +264,9 @@ func validateEventArg(policyName, eventName, argName string) error {
 	return errfmt.Errorf("policy %s, event %s does not have argument %s", policyName, eventName, argName)
 }
 
+// todo: check if we really need this here? it will be tested later when creating policies anyway
+// also - we don't check for validity of args and retval - so why should we check this?
+// also - having this here is a duplication of checking the event flag - so let's do it in one place
```

Comment:
This was a todo for myself.
If you think it is irrelevnat, let's remove this comment

Reply:
I left that to ask @josedonizetti what he thinks.

Reply:
let's remove it :)